src/components/Menu.jsx

```diff
@@ -56,6 +59,14 @@ export default class Menu extends Component {
     }
   }
 
+  setSelectedIndexToLast() {
+    const {
+      options,
+    } = this.props;
+
+    this.selectedIndex = options.length - 1;
+  }
+
```

Comment:
This seems too specific, and I think we can combine it with the existing focus method. So focus(itemIndex). If itemIndex is undefined, focus does what it does now. If an integer, focus and select that item. Allow a negative integer to index from the end -- focus(-1) would mean to select the last item.